src/lint/practice_exercises.nim

```diff
@@ -0,0 +1,64 @@
+import std/[json, os]
+import ".."/helpers
+import "."/validators
+
+proc isValidAuthorOrContributor(data: JsonNode, context: string, path: string): bool =
+  if isObject(data, context, path):
+    result = true
+    if not checkString(data, "github_username", path):
+      result = false
+    if not checkString(data, "exercism_username", path, isRequired = false):
+      result = false
+
+proc checkFiles(data: JsonNode, context, path: string): bool =
```

Comment:
This code is virtually identical to the corresponding concept exercise code, except for `exemplar` being named `example` here. We could extract this functionality into a shared helper.